src/UE5.4/Edgegap/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+
+RUN apt-get update
+RUN apt-get install openssh-server sudo jq -y && \
```

Comment:
@deveshmishrabetide are you sure `sudo` should be here?

Reply:
That is what is used by the older versions. I took your changes for those two files : 

https://github.com/edgegap/edgegap-unreal-plugin/blob/main/src/UE5.0/Edgegap/Dockerfile

Reply:
To add, it's same on 5.3 too: 

https://github.com/edgegap/edgegap-unreal-plugin/blob/main/src/UE5.3/Edgegap/Dockerfile

Reply:
okay I might open another PR later to remove this, thanks for the note!